lib/testium-driver-wd.js

```diff
@@ -164,6 +165,10 @@ function initDriver(testium) {
     }
   );
 
+  wd.addPromiseChainMethod('getChromeDevtoolsPort', () =>
```

Comment:
Could you make this a proper function? Using an arrow function in this position won't get a nice `fn.name`/stack trace id. See `switchToDefaultWindow`/`loadPage` above.

Reply:
I did give function name initially, but linting complained to use arrow method. Should I use disable line to ignore that?